nss-fumbling-penguins/nutshell
# More styles

## Description of Proposed Changes

- 

## Steps to Test

Outline the steps to test

```sh
git fetch --all
git checkout moreStyles

```
make sure that the dashboard appears properly on screen

## Impacted Areas in Application

List general components of the application that this PR will affect:

-login and signup forms styled
dashboard styled
*
